exonum/src/events/noise/tests.rs

```diff
@@ -25,19 +26,23 @@ use tokio_io::{AsyncRead, AsyncWrite};
 
 use std::{net::SocketAddr, thread, time::Duration};
 
+use bytes::BytesMut;
```

Comment:
This is an external crate using, so it should be moved before std usings. Additionally, the empty line before `snow` using should be removed.